src/client/src/Domains/Student/api/index.ts

```diff
@@ -50,6 +50,7 @@ export interface IStudentPreview {
     };
     department?: _IDepartment;
     classStanding?: classStandingTypes;
+    date?: Date;
```

Comment:
The date property of a job application is never undefined in the database, so this property shouldn't be optional

Reply:
But I think the student search result is using the same component and they don't have date?

Reply:
got it